src/validation/Validator.ts

```diff
@@ -15,7 +16,7 @@ export class Validator {
     // Private Properties
     // -------------------------------------------------------------------------
 
-    private validatorJs = require("validator");
+    private validatorJs = validator;
```

Comment:
I am not sure about this line. We can update this later or maybe use validator.js as peer dependency but I have to check more info about it. Maybe we should use for now:
```
private validatorJs: ValidatorJS.ValidatorStatic = require("validator");
```